src/shogun/machine/IterativeLinearMachine.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+* This software is distributed under BSD 3-clause license (see LICENSE file).
+*
+* Authors: Shubham Shukla
+*/
+
+
+#include <shogun/lib/config.h>
+
+#include <shogun/lib/common.h>
+#include <shogun/machine/IterativeLinearMachine.h>
+
+using namespace shogun;
+
+CIterativeLinearMachine::CIterativeLinearMachine() : CLinearMachine()
+{
+ m_current_iteration = 0;
+ m_complete = false;
+ register_params();
+}
+
+CIterativeLinearMachine::~CIterativeLinearMachine()
+{
```

Comment:
SG_UNREF(m_features) (once those are added)

Reply:
how about we just use the ```m_features``` member already present in LinearMachine ?

Reply:
no, pls use another member. We want to move towards not storing features in other classes. Call it `m_continue_features` or so